lib/challenge.js

```diff
@@ -65,7 +69,9 @@ function prompt3() {
  */
 
 function prompt4() {
-  // YOUR CODE HERE
+  for(let i=10; i>0; i--) {
+    console.log(i)
+  }
 }
```

Comment:
Watch the spacing inside the control statement for the for loop on line 72.  To make it easier to read I would rewrite it like:
`for (let i = 10; i > 0; i++)`